user.rb

```diff
@@ -8,7 +9,8 @@ def initialize(name)
     @projects = []
   end
 
-  def add_project(project)
+  def add_project(project, user, role)
```

Comment:
i don't think you need to pass a user to this method, given you're already in the user class

Reply:
should i use self instead?

Reply:
yeah exactly

Reply:
i updated this :)